api.py

```diff
@@ -25,18 +23,13 @@ def __init__(self, service, sql, config):
         self.filename = f"data/{self.classname().lower()}.json"
         self.columns = []
         self.date_columns = []
+        self.rename_columns = {}
         self.request_key = None
         self.table_name = f"GoogleClassroom_{self.classname()}"
 
     def return_all_data(self):
         """Returns all the data in the associated table"""
-        try:
-            return pd.read_sql_table(
-                self.table_name, con=self.sql.engine, schema=self.sql.schema
-            )
-        except InvalidRequestError as error:
-            logging.debug(error)
-            return None
+        return self.sql.read_table(self.table_name)
```

Comment:
So per the conversation below, long term we should keep a try/except here, and then at the call site to return_all_data check for null values before proceeding.

That said, we don't do this today (it just assumes success where this is called), so it's a fix I can make in the other repo first and we can eventually port here.

tl;dr: we can leave this as is, but perhaps fix it later to support graceful failing and give more information to the eventual user.